pkg/testutil/testutil.go

```diff
@@ -430,6 +438,15 @@ func (c *Cmd) AssertOutNotContains(s string) {
 	})
 }
 
+func (c *Cmd) AssertErrNotContains(s string) {
```

Comment:
Hey @Shubhranshu153 

Looks like this is testing stdout (`c.AssertOutWithFunc`) and not stderr.
Can you clarify if the intention here was to test stderr?